go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/robfig/cron v0.0.0-20180505203441-b41be1df6967
 	github.com/stretchr/testify v1.5.1
+	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
```

Comment:
> Is there any particular reason the tag here is from last Summer? Is that when you started this implementation? 

In fact, yes, this PR has been in the works for well over a year, been open for public review since March 28, has gone through an internal SDL process, and had its design and implementation looked over by professional cryptographers.
However, I have bumped the version using "go get -u golang.org/x/crypto"

> Also does dependabot grok these versions? If not, how do we know if there have been fixes to the hkdf package being used?

Inspection of https://github.com/golang/crypto will in fact show that there are no release tags ever dropped on that repository.  This is outside of my control.  However, it is generally a good idea with crypto to not update the code casually due to the risk of changing the output.